apps/teams-test-app/src/components/privateApis/FilesAPIs.tsx

```diff
@@ -164,6 +164,39 @@ const CopyMoveFiles = (): ReactElement =>
     },
   });
 
+const GetFileDownloads = (): ReactElement =>
+  ApiWithoutInput({
+    name: 'getFileDownloads',
+    title: 'Get File Downloads',
+    onClick: {
+      withCallback: setResult => {
+        const callback = (error?: SdkError, files?: files.IFileItem[]): void => {
+          if (error) {
+            setResult(JSON.stringify(error));
+          } else {
+            setResult(JSON.stringify(files));
+          }
+        };
+
+        files.getFileDownloads(callback);
+      },
+      withPromise: async () => {
+        await files.getFileDownloads();
+        return 'files.getFileDownloads complete';
```

Comment:
Sorry I didn't catch this before, but since the resolved promise provides the files.IFileItem[] I wonder if we could display that for the withPromise onClick too?